### PR TITLE
Fix GNSS timestamps

### PR DESCRIPTION
**Pull request by aleks-spes**
There are multiple fixes for timestamps based on the information from RINEX files:
**CHAT**: 
```
zgrep "TIME OF FIRST OBS" chat0870.02o.gz 
  2002     3    28     0     9   30.0000000     GPS         TIME OF FIRST OBS
```
**AKTO**
```
zgrep "TIME OF FIRST OBS" akto1170.06o.gz 
  2006     4    27     0     0    0.0000000     GPS         TIME OF FIRST OBS
```
**RGAR**
```
zgrep "TIME OF FIRST OBS" rgar3110.07o.gz 
  2007    11     7     2     8   30.0000000     GPS         TIME OF FIRST OBS
```
**KTIA**
```
zgrep "TIME OF FIRST OBS" ktia2970.07o.gz 
  2007    10    24     1     0    0.0000000     GPS         TIME OF FIRST OBS
```
**MAHO**
```
zgrep "07  2 20" maho0510.07o
 07  2 20  4 59 30.0000000  0 10G28G29G 8G13G17G27G19G26G11G 3
 07  2 20 23  0  0.0000000  0 10G31G25G 7G16G 1G 6G30G20G14G23
```
**METH**
```
zgrep "TIME OF FIRST OBS" meth2850.10o.gz 
  2010    10    12     0     0    0.0000000     GPS         TIME OF FIRST OBS
 ```
**VEXA**
```
zgrep "TIME OF FIRST OBS" vexa0720.09o.gz 
  2009     3    13     0     0   30.0000000     GPS         TIME OF FIRST OBS
```
**PILK**
```
zgrep "TIME OF FIRST OBS" pilk0710.09o.gz 
  2009     3    12     0     1   30.0000000     GPS         TIME OF FIRST OB
```
**CAST**
```
zgrep "TIME OF FIRST OBS" cast0290.09o.gz 
  2009     1    29     0    38    0.0000000     GPS         TIME OF FIRST OBS
```
**UTK2**
```
zgrep "TIME OF FIRST OBS" utk22040.08o.gz 
  2008     7    22     0     0   30.0000000     GPS         TIME OF FIRST OBS
```
**UTK3**
```zgrep "TIME OF FIRST OBS" utk32000.08o.gz 
  2008     7    18     0     0   30.0000000     GPS         TIME OF FIRST OBS
```
**MCNL**
```
zgrep "TIME OF FIRST OBS" mcnl1480.08o.gz 
  2008     5    27     0     0   30.0000000     GPS         TIME OF FIRST OBS
```
**RAHI**
```
zgrep "TIME OF FIRST OBS" rahi1350.08o.gz 
  2008     5    14     5    20   30.0000000     GPS         TIME OF FIRST OBS
```

### CODE REVIEW

**Review by elidana — CHANGES_REQUESTED** [review]
hi @aleks-spes , thanks for that and great catches! 
I have reviewed this w.r.t. raw data in the archive and there are some minor changes  I am suggesting here. Happy to do a second pass review

File: install/antennas.csv [hunk 1]
```diff
@@ -153,25 +153,25 @@ Trimble Navigation Ltd.,TRM41249.00,12682226,RGKA,0.002,0,0,0,2006-05-08T21:25:3
 Trimble Navigation Ltd.,TRM41249.00,12682299,TGWH,0.002,0,0,0,2007-02-06T22:30:00Z,2016-10-12T03:04:18Z
 Trimble Navigation Ltd.,TRM41249.00,12682307,CLIM,0.002,0,0,0,2005-10-18T01:00:00Z,2015-08-20T22:57:38Z
 Trimble Navigation Ltd.,TRM41249.00,12682354,RGRE,0.002,0,0,0,2007-02-08T00:14:00Z,2013-12-16T23:58:00Z
-Trimble Navigation Ltd.,TRM41249.00,12696858,UTK2,0,0,0,0,2008-07-22T02:00:00Z,2016-08-29T01:00:00Z
+Trimble Navigation Ltd.,TRM41249.00,12696858,UTK2,0,0,0,0,2008-07-22T00:00:30Z,2016-08-29T01:00:00Z
```
[the remark this answers is not in the document]
Comment (aleks-spes) [reply install/antennas.csv]: Fixed in https://github.com/GeoNet/delta/pull/2103/commits/71e6e10b27fc6a2dcb381ff848ec48584a50d71a

File: install/antennas.csv [hunk 2]
```diff
@@ -153,25 +153,25 @@ Trimble Navigation Ltd.,TRM41249.00,12682226,RGKA,0.002,0,0,0,2006-05-08T21:25:3
...
-Trimble Navigation Ltd.,TRM41249.00,60187101,THAP,0.002,0,0,0,2008-06-13T01:05:00Z,2017-10-02T01:51:06Z
-Trimble Navigation Ltd.,TRM41249.00,60187538,UTK3,0,0,0,0,2008-07-18T00:10:00Z,2016-04-04T08:00:00Z
-Trimble Navigation Ltd.,TRM55971.00,30203608,KTIA,0.002,0,0,0,2007-10-24T01:10:00Z,2019-09-25T00:43:37Z
+Trimble Navigation Ltd.,TRM41249.00,60187101,THAP,0.002,0,0,0,2008-06-13T00:28:30Z,2017-10-02T01:51:06Z
+Trimble Navigation Ltd.,TRM41249.00,60187538,UTK3,0,0,0,0,2008-07-18T00:00:30Z,2016-04-04T08:00:00Z
+Trimble Navigation Ltd.,TRM55971.00,30203608,KTIA,0.002,0,0,0,2007-10-24T01:00:00Z,2019-09-25T00:43:37Z
 Trimble Navigation Ltd.,TRM55971.00,30203658,KORO,0.002,0,0,0,2007-06-14T23:55:00Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM55971.00,30203663,NMAI,0.002,0,0,0,2008-02-13T03:05:00Z,9999-01-01T00:00:00Z
-Trimble Navigation Ltd.,TRM55971.00,30203671,MCNL,0.002,0,0,0,2008-05-27T00:05:00Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRM55971.00,30203671,MCNL,0.002,0,0,0,2008-05-27T00:00:30Z,9999-01-01T00:00:00Z
```
[the remark this answers is not in the document]
Comment (aleks-spes) [reply install/antennas.csv]: Fixed in https://github.com/GeoNet/delta/pull/2103/commits/7aa6fb1974cb2c43fdeb7d381155c437d7b1f3f3

File: install/antennas.csv [hunk 3]
```diff
@@ -153,25 +153,25 @@ Trimble Navigation Ltd.,TRM41249.00,12682226,RGKA,0.002,0,0,0,2006-05-08T21:25:3
...
 Trimble Navigation Ltd.,TRM55971.00,30212695,OTAK,0.002,0,0,0,2007-10-06T07:59:00Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM55971.00,30212868,MATW,0.055,0,0,0,2008-07-28T00:01:00Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM55971.00,30213984,MANG,0.055,0,0,0,2007-06-20T03:25:00Z,9999-01-01T00:00:00Z
-Trimble Navigation Ltd.,TRM55971.00,30260173,VEXA,0.055,0,0,0,2009-03-13T04:05:01Z,2019-01-30T20:24:29Z
-Trimble Navigation Ltd.,TRM55971.00,30260375,PILK,0.055,0,0,0,2009-03-12T01:05:01Z,2017-03-14T02:08:46Z
+Trimble Navigation Ltd.,TRM55971.00,30260173,VEXA,0.055,0,0,0,2009-03-13T00:00:30Z,2019-01-30T20:24:29Z
```
[the remark this answers is not in the document]
Comment (aleks-spes) [reply install/antennas.csv]: Fixed in https://github.com/GeoNet/delta/pull/2103/commits/af51fb028e5f8eac87e908b9cacc53916eaa2c43

**Review by elidana — APPROVED** [review]
looks good, thanks @aleks-spes !

**Pull request merged**